Emby.Server.Implementations/Dto/DtoService.cs

```diff
@@ -170,6 +170,11 @@ private static IList<BaseItem> GetTaggedItems(IItemByName byName, User user, Dto
 
         private BaseItemDto GetBaseItemDtoInternal(BaseItem item, DtoOptions options, User user = null, BaseItem owner = null)
         {
+            if (item == null)
```

Comment:
Ideally, null shouldn't ever be passed to this function, but be caught beforehand

Reply:
Hi @Bond-009, that's true, I'll take a look at why this is being sent as null, but it would be good to have this validation to avoid the system crashing in any future changes that can make this to be null again(After fixed).

Reply:
The tests should catch those regressions